### DISCUSSION TITLE
Support for more Vector DBs

### DISCUSSION THREAD

**Discussion by dhruv-anand-aintech** [AI-Northstar-Tech/vector-io#38] [Polls]
Which VectorDB would you like to have supported in Vector IO?

**Comment by dhruv-anand-aintech** [comment]
Marqo (upvote to request)

**Comment by Ronoh4** [comment]
I would love to see ActiveLoop's deeplake included in the supported list

**Comment by dhruv-anand-aintech** [comment]
LanceDB

**Comment by dhruv-anand-aintech** [comment]
ActiveLoop DeepLake

**Comment by dhruv-anand-aintech** [comment]
[Done] Datastax AstraDB

**Comment by asener1** [comment]
When can we have version of Vector IO for Azure AI Search that we can test?

**Reply by dhruv-anand-aintech** [reply]
Hi @asener1, I had reached out to the Azure AI Search team to see if they can contribute an implementation for it. They said they don't have bandwidth to do it at the moment.

I'm currently trying to figure out how I can decide on priority for various DB integrations. The votes here are one of the ways I'm planning to prioritize.

**Reply by farzad528** [reply]
@asener1 Ack, I'm owning the Azure AI Search integration.

**Comment by ddmitov** [comment]
LanceDB

**Reply by dhruv-anand-aintech** [reply]
Thanks for the suggestion! LanceDB export is coming soon

**Reply by dhruv-anand-aintech** [reply]
LanceDB is fully supported now!

**Reply by ddmitov** [reply]
Thank you!

**Comment by ddiddi** [comment]
Neo4j

**Comment by redblackeagleman** [comment]
@dhruv-anand-aintech I see you already did some work on pgvector #54 and #100 and wanted to upvote.

**Comment by Chinoman10** [comment]
Cloudflare (yes, [really](https://developers.cloudflare.com/vectorize/)).
Our #1 slowest step in a RAG we've built. Unforgivable.
